internal/blueprint/disk_customizations.go

```diff
@@ -0,0 +1,356 @@
+package blueprint
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/osbuild/images/pkg/datasizes"
+)
+
+type DiskCustomization struct {
```

Comment:
See my question above, I really would like to avoid this duplication, I'm worried it will lead to drift and I am also not sure why we need it (but maybe/surely I'm missing something here?)